server/test/controllers/admin/AdminProgramTranslationsControllerTest.java

```diff
@@ -299,6 +298,8 @@ public void update_outOfSyncFormData_redirectsToFreshData() throws Exception {
                     .put("localized-email-1", "updated spanish second status email")
                     .put("application-step-title-0", "step one title")
                     .put("application-step-description-0", "step one description")
+                    .put("application-step-title-1", "step two title")
```

Comment:
Two application steps are built into this test program so this test was failing without the translation for the second one.